### DISCUSSION TITLE
Does `prisma-binding-argument-transform` exists?

### DISCUSSION THREAD

**Discussion by braska** [prisma/prisma#2814] [Q&A] [answered]
I saw some mentions about `prisma-binding-argument-transform` package in docs here: https://www.prisma.io/docs/guides/upgrade-from-prisma-1/upgrading-prisma-binding-to-sdl-first#31-migrate-graphql-queries

But looks like there no such package in NPM. Is it outdated documentation? Or opposite this documentation is ahead and there is only plans to publish such package?

**Comment by braska** [comment, marked as answer]
I just created issue: https://github.com/prisma/docs/issues/491

**Comment by ludralph** [comment]
Hi @braska,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
